homework/tai_nguyen/1209_remove_all_adjacent_duplicates_in_string_ii.ts

```diff
@@ -0,0 +1,35 @@
+interface StackElement {
+  value: string;
+  count: number;
+}
+
+function getTop<T>(stack: T[]) {
+  if (stack.length) {
+    return stack[stack.length - 1];
+  }
+
+  return undefined;
+}
```

Comment:
we can use conditional operator to make it shorter
```suggestion
function getTop<T>(stack: T[]) {
 return stack.length ? stack[stack.length - 1] : undefined;
}
```

Reply:
Thanks @tadashii1417, I think it is another concise approach!